dev/ProgressBar/ProgressBar.cpp

```diff
@@ -159,10 +170,11 @@ void ProgressBar::UpdateWidthBasedTemplateSettings()
             return std::make_tuple(0.0f, 0.0f);
         }();
 
-        const double indicatorWidthMultiplier = -0.4;
+        templateSettings->ContainerAnimationStartPosition(width * -0.4);
```

Comment:
-1.0 per spec

Reply:
also need mid value and end value for both indicators as they are different

Reply:
I followed the specs exactly as outlined in this [codepen](https://codepen.io/anon/pen/aQeVOm). The translation value for `"bar-infinite-a"` is `-100%`, where the percentage signifies percentage width of the indicator, not the progress bar. As the indicator is spec'd at 40% width of the progress bar, the -0.4 is the correct multiplier.